webapp/src/modules/bid/utils.js

```diff
@@ -0,0 +1,21 @@
+export const getUniqueAssetBidIds = (bidIds, allBids) => {
+  const assetIds = {}
+  return bidIds.reduce((ids, id) => {
+    const bid = allBids[id]
+
+    if (bid) {
+      const assetKey = `${bid.asset_type}_${bid.asset_id}`
+      if (!assetIds[assetKey]) {
+        assetIds[assetKey] = true
+        return [...ids, id]
+      }
+    }
+    return ids
+  }, [])
```

Comment:
I think this method is trying to replicate a `Set` by using an object as cache. I think it'll look waay simpler just using the language structure